bin/setup-faxws.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+
+echo "Setting up the Fax server"
+
+## 3. Clone
+git clone https://dennis_warren@bitbucket.org/openoscar/faxws.git
+
+## 4. Checkout
+git checkout origin/release
+
+# 5. current MySQL pass and username
+echo "Setting Fax WS database passwords... "
+
+read -p "MySQL database username? " username
+read -p "MySQL database password? " passwd
+
+perl -pi -e "s/{user}/$username/"  ./src/main/webapp/META-INF/context.xml
+perl -pi -e "s/{passwd}/$passwd/"  ./src/main/webapp/META-INF/context.xml
+
+# 6. new FaxWs pass and username
+read -p "What username do you want to log in as in tomcat? " tomcatUser
```

Comment:
Does the fax WS have a little UI like expedius? How are these creds used? By an actual person? For doing what operations?

Reply:
No GUI. This is a standard configuration for the Tomcat Database Realm Authentication. Unlike Expedius, this authentication locks down the FaxWs web services.  
https://tomcat.apache.org/tomcat-7.0-doc/realm-howto.html

Reply:
What system actually uses these creds to access FaxWS?

Reply:
OSCAR

Reply:
The account details are stored in an OSCAR db table.  I suppose this data could be inserted into the OSCAR db during install.  OSCAR <--> FaxWs <--> SRFax